tests/cypress/support/pages/Homepage.js

```diff
@@ -1,76 +1,56 @@
-const logoAtTopLeft = 'img[alt="QMR Logo"]';
-const loginButton = "a#loginButton";
-const apsSubmissionAppTxt = "//div[@class='Home']//h1[.='APS Submission App']";
-const sentence = '(//div[@class="footer-fed-gov-text"])[1]';
+const appLogo = "//img[@alt='QMR Logo']";
+const headerAppLogo = 'img[alt="QMR Logo"]';
+const headerLoginButton = "a#loginButton";
+const footerAddress = '(//div[@class="footer-wrapper"]/div)[2]';
+const footerEmail = ".footer-email";
+const footerHhsLogo = "img[alt='Department of Health and Human Services logo']";
+const footerMdctLogo = "//img[@alt='Mdct logo']";
+const footerMedicaidLogo = "//img[@alt='Medicaid.gov logo']";
 const medicaidLogo = "img[alt='Medicaid.gov logo']";
-const emailBottomLeft = ".footer-email";
-const federalLogo = "img[alt='Department of Health and Human Services logo']";
-const addressBottomRight = '(//div[@class="footer-wrapper"]/div)[2]';
-const coreSetMeasuresReportingTitle =
-  "//h2[contains(text(),'FFY 2021 Core Set Measures Reporting')]";
-const adultCoreSetMeasures = "//a[ contains(@href, 'ACS') ]";
-//oy2-15212 qmr mdct logo
-const qualityMeasureReportingLogo = "//img[@alt='QMR Logo']";
-const mdctLogo = "//img[@alt='Mdct logo']";
-const medicaidLogoBottom = "//img[@alt='Medicaid.gov logo']";
+const footerSentence = '(//div[@class="footer-fed-gov-text"])[1]';
 
 export class Homepage {
   launch() {
     cy.visit("/");
   }
 
-  verifyQMRMDCTMedicaidLogoAtHomePage() {
-    cy.xpath(qualityMeasureReportingLogo).should("be.visible");
-    cy.xpath(mdctLogo).should("be.visible");
-    cy.xpath(medicaidLogoBottom).should("be.visible");
-  }
-
-  verifyTheTitleCoreSetMeasureReporting() {
-    cy.xpath(coreSetMeasuresReportingTitle).should("be.visible");
-  }
-
-  clickAdultCoreSetMeasures() {
-    cy.wait(3000);
-    cy.scrollTo("top");
-    cy.xpath(adultCoreSetMeasures).click({ force: true });
-    //cy.xpath(adultCoreSetMeasures).click();
+  verifyLogos() {
+    cy.xpath(appLogo).should("be.visible");
+    cy.xpath(footerMDCTLogo).should("be.visible");
```

Comment:
This should be `footerMdctLogo` (or capitalized on line 8)

Reply:
Good catch, thanks.